h2o-hadoop/scripts/docker/common/start_h2o

```diff
@@ -0,0 +1,13 @@
+#! /bin/bash
+
+set -e
+
+if [[ $(echo ${START_H2O} | tr -s '[:upper:]' '[:lower:]') == 'true' ]]; then
```

Comment:
Same as above. Should not be part of the image. Hadoop images should create a suitable environment to run the job but should not generally not be aware of H2O.